### DISCUSSION TITLE
How to configure alerts (alertng) without GUI ?

### DISCUSSION THREAD

**Discussion by mcalteau** [grafana/grafana#37258] [Alerting]
I'm using helm charts to create my grafana env. I can set datasources and dashboard in configuration but I don't find any informations to configure alert rules/contact points/... the same way.

We have many alertrule with multiple requests and it's very painfull to copy all the configuation on multiple environments.

**Comment by pkolyvas** [comment]
Hey there!

Thanks for taking the time to write us. The best place to ask about workflow question and solicit input/feedback is [on our community forums.](https://community.grafana.com/c/support/33)

**Reply by Oliver-Sellwood** [reply]
Hey, seems like this is a missing or undocumented feature and as I understand feature requests should be directed here so not sure the community forums is the go to and it also seems to have been [asked there](https://community.grafana.com/t/provision-alerts/13334) (though before the release of unified logging).

Additionally this question has come up several times in the slack #grafana channel of the community slack channel.

Could you clarify whether it is possible to provision unified alerting and point to some docs or if there is something I missed in the community forum maybe point me there as I'm coming up blank in all these sources.

Also thanks for all the hardwork on Grafana!!

**Reply by Slach** [reply]
@pkolyvas i aslo tried to find how to provision unified alerts

**Reply by peterholmberg** [reply]
Hello! We do not have any support for provisioning alerts currently but it is on our roadmap to solve this.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
